theano/sandbox/rng_mrg.py

```diff
@@ -1518,6 +1521,7 @@ def multinomial_wo_replacement(self, size=None, n=1, pvals=None,
         warnings.warn('MRG_RandomStreams.multinomial_wo_replacement() is '
                       'deprecated and will be removed in the next release of '
                       'Theano. Please use MRG_RandomStreams.choice() instead.')
+        assert size is None
         return self.choice(size=n, a=None, replace=False, p=pvals,
                            dtype=dtype, nstreams=nstreams, ndim=ndim)
```

Comment:
As far as I understand, the value of a is completely ignored here.  We should at least check that it's two.